src/lib/eina/eina_sched_posix.c

```diff
@@ -118,4 +118,5 @@ eina_sched_prio_init(Eina_Thread_Call* c)
         if (params.sched_priority > max) params.sched_priority = max;
         pthread_setschedparam(self, pol, &params);
      }
+
```

Comment:
ué?

Reply:
Ops

Reply:
is it from master?

Reply:
Nope :D It was _my bad_ while rebasing - then - copying - pasting - then - checking.

Reply:
So remove it please :D

Reply:
> So remove it please :D

FORCE ME >:D

Reply:
Oh you did it damn you're so convincing.